### PR TITLE
Add `CONTRIBUTING.md` file

### LINKED ISSUE #3: Add contribution Guide
Create a file name **contribution.md** and use this [template.](https://github.com/faradayafrica/Faraday-community/blob/main/contributing.md) 
Modify the link to the readme to point to the README file for this repo.

### PR DESCRIPTION
**Pull request by mayankshukla94**
Fixes https://github.com/faradayafrica/Url-Shortener-API/issues/3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# What do I need to know to help?
+
+If you are looking to help with a code contribution, our project uses Python. If you don't feel ready to make a code contribution yet, no problem! There may be some open issues related to documentation or design soon so we recommend that you watch the repo to get notified when there are new activities happening on the project. You can also propose improvements by opening a new issue.
+
+## How do I make a contribution?
+
+If you've never made an open source contribution before or you are wondering how contributions work in our project? Here's a quick rundown!
+
+1. Fork the repository to your account. This means that you will have a copy of the repository under `<your-GitHub-username>/<repository-name>`.
```
Comment (devgenix) [inline CONTRIBUTING.md:9]: Hi, please modify the repository link to point to the repository.

Comment (mayankshukla94) [reply CONTRIBUTING.md:9]: Could you please elaborate? I did not understand the changes requested

Comment (mayankshukla94) [reply CONTRIBUTING.md]: I have done the changes. Please check and let me know if that is what you requested.

Comment (devgenix) [reply CONTRIBUTING.md]: Sorry I didn't respond in time.

**Pull request merged**